core/internal/cltest/factories.go

```diff
@@ -243,66 +243,66 @@ func MustInsertUnconfirmedEthTxWithInsufficientEthAttempt(t *testing.T, borm txm
 	attempt.SignedRawTx = rlp.Bytes()
 
 	attempt.State = txmgrtypes.TxAttemptInsufficientEth
-	require.NoError(t, borm.InsertEthTxAttempt(&attempt))
-	etx, err := borm.FindEthTxWithAttempts(etx.ID)
+	require.NoError(t, txStorageService.InsertEthTxAttempt(&attempt))
+	etx, err := txStorageService.FindEthTxWithAttempts(etx.ID)
 	require.NoError(t, err)
 	return etx
 }
 
 func MustInsertConfirmedMissingReceiptEthTxWithLegacyAttempt(
-	t *testing.T, borm txmgr.ORM, nonce int64, broadcastBeforeBlockNum int64,
-	broadcastAt time.Time, fromAddress common.Address) txmgr.EthTx {
+	t *testing.T, txStorageService txmgr.EvmTxStorageService, nonce int64, broadcastBeforeBlockNum int64,
+	broadcastAt time.Time, fromAddress common.Address) txmgr.EvmTx {
 	etx := NewEthTx(t, fromAddress)
 
 	etx.BroadcastAt = &broadcastAt
 	etx.InitialBroadcastAt = &broadcastAt
 	etx.Nonce = &nonce
 	etx.State = txmgr.EthTxConfirmedMissingReceipt
-	require.NoError(t, borm.InsertEthTx(&etx))
+	require.NoError(t, txStorageService.InsertEthTx(&etx))
 	attempt := NewLegacyEthTxAttempt(t, etx.ID)
 	attempt.BroadcastBeforeBlockNum = &broadcastBeforeBlockNum
 	attempt.State = txmgrtypes.TxAttemptBroadcast
-	require.NoError(t, borm.InsertEthTxAttempt(&attempt))
+	require.NoError(t, txStorageService.InsertEthTxAttempt(&attempt))
 	etx.EthTxAttempts = append(etx.EthTxAttempts, attempt)
 	return etx
 }
 
-func MustInsertConfirmedEthTxWithLegacyAttempt(t *testing.T, borm txmgr.ORM, nonce int64, broadcastBeforeBlockNum int64, fromAddress common.Address) txmgr.EthTx {
+func MustInsertConfirmedEthTxWithLegacyAttempt(t *testing.T, txStorageService txmgr.EvmTxStorageService, nonce int64, broadcastBeforeBlockNum int64, fromAddress common.Address) txmgr.EvmTx {
 	timeNow := time.Now()
 	etx := NewEthTx(t, fromAddress)
 
 	etx.BroadcastAt = &timeNow
 	etx.InitialBroadcastAt = &timeNow
 	etx.Nonce = &nonce
 	etx.State = txmgr.EthTxConfirmed
-	require.NoError(t, borm.InsertEthTx(&etx))
+	require.NoError(t, txStorageService.InsertEthTx(&etx))
 	attempt := NewLegacyEthTxAttempt(t, etx.ID)
 	attempt.BroadcastBeforeBlockNum = &broadcastBeforeBlockNum
 	attempt.State = txmgrtypes.TxAttemptBroadcast
-	require.NoError(t, borm.InsertEthTxAttempt(&attempt))
+	require.NoError(t, txStorageService.InsertEthTxAttempt(&attempt))
```

Comment:
Quite a verbose name here, you might consider txORM or txss or something

Reply:
I agree, although I think we're trying to phase out 'orm'. On another thread we brainstormed a few ideas and thought `txStore`/`txmStore` made sense. Whatever we choose, we should try to be consistent.

Reply:
+1 to `txStore` nice and concise

Reply:
Can we please do the renames in this PR? There is no need to thrash this around for everyone doing rebases when it can be renamed easily now.